docs/examples.md

```diff
@@ -478,7 +477,7 @@ Note that the ability to set build environment variables is currently in beta.
 
 ## LocalNuGetPackageFunction and LocalNuGetPackageCode
 
-The example above describes how to deploy a function with local *project* dependencies.
+The example above describes how to deploy a function with local _project_ dependencies.
```

Comment:
I don't understand why this has been changed. I'd personally suggest using stars for italics instead of underscores, and this change makes the Markdown inconsistent with other docs in the same repo, e.g. docs/deployment.md